code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm

```diff
@@ -198,3 +198,12 @@
 	)
 	result = /obj/item/reagent_containers/food/snacks/salad/ricepork
 	subcategory = CAT_MEAT
+
+/datum/crafting_recipe/food/ribs
+	name = "BBQ Ribs"
+	reqs = list(
+		/datum/reagent/consumable/bbqsauce = 5,
+		/obj/item/reagent_containers/food/snacks/meat/steak/plain = 2
```

Comment:
would be funny if it required bones

Reply:
Yeah, but lets be real, then nobody would ever make it.

Reply:
@ArcaneMusic make it require rods as fake bones

Reply:
Done